fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -789,6 +792,11 @@ showProcStatement
     : SHOW PROC path=string
     ;
 
+// ---------------------------------------- Backup Store Statement -----------------------------------------------------
+showBackupStatement
+    : SHOW BACKUP (FROM identifier)?
```

Comment:
```suggestion
    : SHOW BACKUP ((FROM | IN) identifier)?
```

Reply:
I see that the official document does not have the syntax of "| in", so this has not been modified, and other things have been modified. https://docs.starrocks.com/zh-cn/main/sql-reference/sql-statements/data-manipulation/SHOW%20BACKUP